web/docs/tql2/operators/slice.md

```diff
@@ -0,0 +1,72 @@
+# slice
+
+Keep a range of events within the interval `[begin, end)` stepping by `stride`.
+
+```
+slice [begin=int] [end=uint] [stride=uint]
```

Comment:
```suggestion
slice [begin=int, end=int, stride=int]
```

Note that I also changed `uint` to `int`. This is (A) because these can be negative, but also (B) even if not, we might want to not expose the `int/uint` distinction in operator and function docs. Instead, we should probably use `int` everywhere.

Reply:
Fixed but I think for arguments which cannot be negative, we should specify `uint`.